src/cancelContentPlay.js

```diff
@@ -14,7 +14,7 @@ const cancelContentPlay = function(player) {
   }
 
   // Avoid content flash on non-iPad iOS
-  if (videojs.browser.IS_IOS) {
+  if ((videojs.browser.IS_IOS) && (!videojs.browser.IS_IPAD)) {
```

Comment:
Thanks! I did see it but since the comment said non iPad, I was trying to match the logic to the comment.  Changed it to IS_IPHONE still works on iPad and doesn't break iPhone in IMA3 or FW, so I'll resubmit.